experiments/download_ssl4eo.py

```diff
@@ -627,17 +614,24 @@ def fix_random_seeds(seed: int = 42) -> None:
     else:
         ext_path = os.path.join(args.save_path, "checked_locations.csv")
 
-    # build the grid from existing coordinates
-    grid_dict: Dict[Tuple[int, int], Any] = {}
+    # build the rtree from existing coordinates
```

Comment:
Why are we build two separate R-trees?

Reply:
only one. This part is to resume/initialize from existing sampled coords. The tree is then used for further sampling.

Reply:
Both sections of code insert into an R-tree though?

Reply:
yes the same tree

Reply:
yes that's what i did